docs/analytics.md

```diff
@@ -161,9 +161,16 @@ statistics, etc.). There is currently a reference implementation of a
 ### Setup and Testing
 
 The main testing approach for the /stats page UI is manual testing.  For UI
+<<<<<<< HEAD
```

Comment:
These lines are automatically added by git in case of a merge conflict. You need to remove them. Please refer to our (git-guide)[http://zulip.readthedocs.io/en/latest/git-guide.html] if you need any help on how to resolve a merge conflict.